### ISSUE TITLE
Decouple HvWizard from HvDialog

### ISSUE THREAD

**Issue by andrew-kayne** [lumada-design/hv-uikit-react#2963]
### Latest version

- [X] I have tested the latest version

### No Hitachi Vantara confidential content

- [X] I'm not including any Hitachi Vantara confidential content

### Current behavior 😯

The new HvWizard component is coupled to HvDialog, primarialy for styling reasons. However, our design uses a Wizard outside of a dialog and in some cases, we even have a wizard within another wizard. 

### Expected behavior 🤔

HvWizard should make no assumptions on the context it is being used in. That is, it could be a screen, dialog, drawer or even a page of another wizard.

The Wiard steps, title, content and actions should be styled consistently with their own styling rules/classes so that there is no dependency or relation to other components.

### Steps to reproduce 🕹

None

### Your Environment 🌎

All

**Comment by francisco-guilherme** [comment]
Hi @andrew-cockayne, tks for reaching out.

Although i understand and agree with some of your concerns, the contributed version of the Wizard component is following, as it should, the DS guidelines: https://xd.adobe.com/view/01d6a8c3-6641-4f22-b727-5bf7786eb45a-5485/?fullscreen.

If you have a different use case, please check with your design representatives, to make sure that your requirements are rooted with a valid rationale.

As the current version is the result of an external contribution, please feel free to contribute with your own version as it might help other teams with similar use cases.

**Comment by rubenbarroshv** [comment]
Hello,

I am currently trying out a solution for this. Later today or early next week I will do a draft PR and tag both of you.

**Comment by andrew-cockayne** [comment]
@rubenbarroshv beat me too it. I had spoken to him directly about this. 😀

**Comment by rubenbarroshv** [comment]
Hello,

[Here](https://github.com/rubenbarroshv/hv-uikit-react/pull/1) is the PR with the decoupling. I worked on top of the Summary fix, so I pointed this to my repo to avoid confusion at this stage. When the fix is merged I will point this to uikit.

If any of you have any suggestions I could improve how to use this feature. For now we just have to put the fullscreen flag on the HvWizardContainer and it should work. Added some className props to help the styling of each component.
